Apatira_Authman/fullfriends/server.py

```diff
@@ -0,0 +1,61 @@
+from flask import Flask, render_template, session, redirect, request, flash
+from mysqlconnection import MySQLConnector
+import re
+
+app = Flask(__name__)
+mysql = MySQLConnector(app, 'friends')
+print mysql
+
+@app.route('/')
+def index():
+    selectAll = "SELECT * FROM friends"
+    friends = mysql.query_db(selectAll)
+    return render_template('index.html', friendsList = friends)
+
+@app.route('/process', methods=['POST'])
+def newFriend():
+    insertFriend = "INSERT INTO friends (first_name, last_name, occupation, created_at, updated_at) VALUES (:firstName, :lastName, :occupation, Now(), Now())"
+    if len(request.form['first_name'] ) < 1:
+		flash("Enter Valid Name:")
+		redirect('/')
+	if len(request.form['last_name']) < 1:
+		flash("Enter Valid Name:")
+		redirect('/')
```

Comment:
`redirect('/')` doesn't really do anything. you have to `return redirect('/')` for it to work